### ISSUE TITLE
Filebeat installs successfully but does not start/restart

### ISSUE THREAD

**Issue by jofomah** [elastic/ansible-beats#70]
**Beats product**: Filebeat

**Beats version** 7.5.1

**Role version**:  elastic.beats 7.5.1

**OS version** (`uname -a` if on a Unix-like system): 
- Linux dev.sandbox.example 2.6.32-754.23.1.el6.x86_64 #1 SMP Thu Sep 26 12:05:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
- OS Version: CentOS release 6.10 (Final)

**Description of the problem including expected versus actual behaviour**:
When i install role and run, https://github.com/elastic/ansible-beats/blob/master/tasks/beats-redhat.yml tasks are completed successfully, Filebeat is installed but fails when restart handler is called.

**Playbook**:
```
- name: Install Graylog Collector
  hosts: graylog_clients
  roles:
    - role: elastic.beats
  vars:
    beats_version: 7.5.1
    beat: filebeat
    beat_conf:
      inputs:
        - type: log
          enabled: true
          paths: []
```

**Provide logs from Ansible**:
Find attached a copy of the log
[output.log](https://github.com/elastic/ansible-beats/files/4108620/output.log)

**Beats logs if relevant**:

```
2020-01-23T07:58:49.413-0500	INFO	[publisher]	pipeline/module.go:97	Beat name: dev.designintools.intel.com
2020-01-23T07:58:49.413-0500	INFO	instance/beat.go:402	filebeat stopped.
2020-01-23T07:58:49.422-0500	ERROR	instance/beat.go:916	Exiting: no modules or inputs enabled and configuration reloading disabled. What files do you want me to watch?
```

**Comment by jofomah** [comment]
@jmlrt @gingerwizard @tgadiev Sorry for invoking you guys, please i need help does any one know why?

**Comment by tgadiev** [comment]
@jofomah looks like the empty list in `paths` causes that.

**Comment by jofomah** [comment]
> @jofomah looks like the empty list in `paths` causes that.

I updated to 

```
    beat_conf:
      inputs:
        - type: log
          enabled: true
          paths:
            - '/var/log/messages'
```

and the path is not empty in `/etc/filebeat/filebeat.yml`, yet same outcome.

![Screenshot from 2020-01-28 20-02-15](https://user-images.githubusercontent.com/5929681/73296105-5667bf00-4209-11ea-96cd-dd3fd0ec49ef.png)

**Comment by jofomah** [comment]
@tgadiev I updated `inputs` to  `filebeat.inputs` and it worked.

```
  roles:
    - role: elastic.beats
  vars:
    beats_version: 7.5.1
    beat: filebeat
    beat_conf:
      filebeat.inputs:
        - type: log
          enabled: true
          paths: []
```

**Comment by jmlrt** [comment]
Hi @jofomah and @tgadiev,

The filebeat example in the README was still using old configuration syntax (`inputs:` instead of `filebeat.inputs`). Thank you for reporting this.

This will be fixed in #71
